### PR TITLE
Provide a AWS Policy file for minimum permissions

### PR DESCRIPTION
**Pull request by leorossi**
Refs: #47

### CODE REVIEW

File: docs/quick-start/README.md [hunk 1]
```diff
@@ -80,13 +80,15 @@ Clone your newly created repository and follow the instructions below.
    ```
    __Note:__ `config/dev.json` file is specific to your personal setup and should not be tracked in GIT.
 
-7. Bootstrap AWS CDK on target AWS account and region, e.g.:
+7. If you want to restrict access to your AWS User, attach the policy you find in `aws-policy.json` file.
```
Comment (nigelhanlon) [inline docs/quick-start/README.md:83]: It's not quite clear where to attach this or how. Can you give or reference an example?

**Comment by irelandm** [comment]
This is a partial implementation of #47

File: aws-policy.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+      {
+          "Sid": "VisualEditor0",
```
Comment (irelandm) [inline aws-policy.json:5]: We should give either a random or informative Sid

**Comment by irelandm** [comment]
Archiving this, as superseded by https://github.com/nearform/mira-iam-explorer

**Pull request closed without merging**